examples/hotspot.rs

```diff
@@ -74,12 +72,17 @@ fn main() {
         password,
     } = parse_options();
 
-    let manager = manager::new();
+    let pass_str = match password {
```

Comment:
Good point - this should be handled better. The password is WPA pre-shared key, which is either a 64-characters long hexadecimal or between 8 and 63 ASCII chars. Here is the validation function used in NetworkManager itself: https://github.com/NetworkManager/NetworkManager/blob/master/libnm-core/nm-utils.c#L1160